core/templates/dev/head/pages/exploration_editor/statistics_tab/IssuesService.js

```diff
@@ -36,7 +36,8 @@ oppia.factory('IssuesService', [
     var renderMultipleIncorrectIssueStatement = function(stateName) {
       var statement =
         'Several learners submitted answers to card "' + stateName +
-        '" several times, then gave up and quit.';
+        '" ' + NUM_INCORRECT_ANSWERS_THRESHOLD + ' times, ' +
```

Comment:
@hoangviet1993 Nope the overhead issue statement should not include this constant. The number of multiple incorrect answers need not be 3 always. It could be any value > 3, depending on each playthrough in this issue. I'd suggest reverting back to how it was before.

Reply:
Reverted my changes :+1: